.github/awstest.yml

```diff
@@ -0,0 +1,29 @@
+name: Arcadia-Science/metagenomics AWS tests
+# This workflow can be triggered manually with the GitHub actions workflow dispatch button.
+# It runs the -profile 'test' on AWS batch
+
+on:
+  workflow_dispatch:
+jobs:
+  run-tower:
+    name: Run AWS tests
+    if: github.repository == 'Arcadia-Science/metagenomics'
+    runs-on: ubuntu-latest
+    steps:
+      # Launch workflow using Tower CLI tool action
+      - name: Launch workflow via tower
+        uses: nf-core/tower-action@v3
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}}
+          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/metagenomics/work-${{ github.sha }}
+          parameters: |
+            {
+              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/metagenomics/results-${{ github.sha }}"
```

Comment:
do you want the outdir to have something like `test_` in the name as well? it might help if we start running this workflow a bunch on tower with real data